scripts/main.py

```
@@ -82,17 +82,17 @@ def update(path=''):
+    sheets = mlb_standings.mlbstandings.light_google_wrappers.Spreadsheets(authed_session)
+    base_web = mlb_standings.mlbstandings.web.Web()
     web = AbstractRateLimitedWeb(base_web, SimpleRateLimiter(15))
-    updater = mlbstandings.updater.Updater(d, files, sheets, CONTENTS_SPREADSHEET_ID, web)
+    updater = mlb_standings.mlbstandings.updater.Updater(d, files, sheets, CONTENTS_SPREADSHEET_ID, web)
     # TODO remove once everything works with new versions
     base_web.read('https://www.baseball-reference.com/')
     print('No problem reading www.baseball-reference.com')
     while True:
         status = updater.update()
-        if status == None or status == mlbstandings.updater.SeasonStatus.OVER or not backfill:
+        if status == None or status == mlb_standings.mlbstandings.updater.SeasonStatus.OVER or not backfill:
```

Flake8 (7.2.0)

[error] comparison to None should be 'if cond is None:'

(E711)